tf/modules/monitoring/signup-throttling-alert.tf

```
@@ -0,0 +1,39 @@
+resource "google_logging_metric" "api_signup_throttling_applied" {
+  name        = "${var.cluster_name}-api_signup_throttling_applied"
+  description = "A log based metric that detects when api was throttling sign up attempts"
+
+  # Filter for the secondary pod and look for readiness probe failures
+  filter = "resource.labels.cluster_name=\"${var.cluster_name}\" AND resource.labels.container_name:\"api-\" AND severity=ERROR AND resource.type=\"k8s_container\" AND textPayload:\"WARN_SIGNUP_THROTTLED\""
+
+  metric_descriptor {
+    metric_kind = "DELTA"
+    value_type  = "INT64"
+  }
+}
+
+resource "google_monitoring_alert_policy" "alert_api_signup_throttling_applied" {
+  display_name = "(${var.cluster_name}): Signup was throttled"
+  combiner     = "OR"
+  notification_channels = [
+    "${var.monitoring_email_group_name}"
+  ]
+  documentation {
+    content = "Alert triggers when a user tried to create a wikibase.cloud account, but the configured limits were already exceeded."
+  }
+  conditions {
+    display_name = "(${var.cluster_name}): Signup was throttled"
+    condition_threshold {
+      filter     = "metric.type=\"logging.googleapis.com/user/${google_logging_metric.api_signup_throttling_applied.name}\" AND resource.type=\"k8s_pod\""
```

```suggestion
      filter     = "metric.type=\"logging.googleapis.com/user/${google_logging_metric.api_signup_throttling_applied.name}\" AND resource.type=\"k8s_container\""
```

Looks like this actually needs to be a container too or it fails to apply with:
`Error: Error creating AlertPolicy: googleapi: Error 400: The supplied filter does not specify a valid combination of metric and monitored resource descriptors. The query will not return any time series.`